xds/src/main/java/io/grpc/xds/WeightedRoundRobinLoadBalancer.java

```diff
@@ -145,7 +178,8 @@ public Status acceptResolvedAddresses(ResolvedAddresses resolvedAddresses) {
   @Override
   public SubchannelPicker createReadyPicker(Collection<ChildLbState> activeList) {
     return new WeightedRoundRobinPicker(ImmutableList.copyOf(activeList),
-        config.enableOobLoadReport, config.errorUtilizationPenalty, sequence);
+        config.enableOobLoadReport, config.errorUtilizationPenalty, sequence, getHelper(),
+        rrFallbackCounter);
```

Comment:
Why is `rrFallbackCounter` passed in? It is static. Did I miss a test making use of this?

Reply:
Leftover from a previous approach (that would not have worked anyway). Removed now.